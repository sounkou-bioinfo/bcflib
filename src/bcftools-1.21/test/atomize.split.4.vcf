##fileformat=VCFv4.3
##contig=<ID=11,length=135006516>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
11	100	.	CC	GG	.	.	.
11	200	.	CC	GGGG	.	.	.
11	300	.	C	GGGG	.	.	.
