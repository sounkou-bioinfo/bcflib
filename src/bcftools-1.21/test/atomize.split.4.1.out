##fileformat=VCFv4.3
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=11,length=135006516>
##INFO=<ID=OLD_REC,Number=1,Type=String,Description="Original variant. Format: CHR|POS|REF|ALT|USED_ALT_IDX">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
11	100	.	C	G	.	.	OLD_REC=11|100|CC|GG|1
11	101	.	C	G	.	.	OLD_REC=11|100|CC|GG|1
11	200	.	C	G	.	.	OLD_REC=11|200|CC|GGGG|1
11	201	.	C	CGG	.	.	OLD_REC=11|200|CC|GGGG|1
11	201	.	C	G	.	.	OLD_REC=11|200|CC|GGGG|1
11	300	.	C	CGGG	.	.	OLD_REC=11|300|C|GGGG|1
11	300	.	C	G	.	.	OLD_REC=11|300|C|GGGG|1
